in/index.html.mako

```diff
@@ -77,6 +77,10 @@
           data-text-off="Compare Selected">
           Compare Selected
         </button>
+
+        <button class="btn btn-warning btn-clear">
```

Comment:
I opted to just change it to `btn-primary`, as conditionally disabling it based on whether or not url or datatable params had changed from init was relatively expensive to do with little gain IMO. Ready for review